coala_quickstart/generation/Bears.py

```diff
@@ -0,0 +1,54 @@
+from pyprint.NullPrinter import NullPrinter
+
+from coala_quickstart.Strings import BEAR_HELP
+from coala_utils.Question import ask_question
+from coalib.settings.ConfigurationGathering import (
+    load_configuration, get_filtered_bears)
+from coalib.misc.DictUtilities import inverse_dicts
+from coalib.collecting.Collectors import (
+    collect_all_bears_from_sections, filter_section_bears_by_languages)
+from coalib.output.printers.LogPrinter import LogPrinter
+
+
+def filter_relevant_bears(used_languages):
+    """
+    From the bear dict, filter the bears per relevant language.
+
+    :param used_languages:
+        A list of tuples with language name as the first element
+        and percentage usage as the second element; sorted by
+        percentage usage.
+    :return:
+        A dict with language name as key and bear classes as value.
+    """
+    log_printer = LogPrinter(NullPrinter())
+    used_languages.append(("All", 100))
+
+    result = {lang: set(inverse_dicts(*get_filtered_bears(
```

Comment:
that's actually not a result, right? Can you try to find a better name? Like lang_bear_dict or so? (Is it a language-bear dict? I don't know :P)